rules/60-fido-id.rules

```diff
@@ -0,0 +1,7 @@
+# do not edit this file, it will be overwritten on update
+
+ACTION=="remove", GOTO="fido_id_end"
```

Comment:
Shouldn't this also say `SUBSYSTEM="hidraw"` to not match on every single device and needlessly execute a goto that does nothing?

Reply:
I checked a handful of randomly picked rules files and they are all start with the same rule (modulo the goto name). So presumably it's not a problem?

Reply:
Well, they're all inefficient in the same way, in my opinion. But the impact is probably negligible, so let's leave it as it is.